GTG/gtk/browser/main_window.py

```diff
@@ -1282,45 +1282,50 @@ def on_dismiss_task(self, widget=None):
                 task.set_status(Task.STA_DISMISSED)
                 self.close_all_task_editors(uid)
 
-    def apply_filter_on_panes(self, filter_name, refresh=True, parameters=None):
-        """ Apply filters for every pane: active tasks, closed tasks """
-
-        for pane in self.vtree_panes:
-            vtree = self.req.get_tasks_tree(name=pane, refresh=False)
-            vtree.apply_filter(filter_name, refresh=refresh, parameters=parameters)
-
-    def unapply_filter_on_panes(self, filter_name, refresh=True):
-        """ Apply filters for every pane: active tasks, closed tasks """
-        for pane in self.vtree_panes:
-            vtree = self.req.get_tasks_tree(name=pane, refresh=False)
-            vtree.unapply_filter(filter_name, refresh=refresh)
+    def _reapply_filter(self, current_pane):
+        filters = self.get_selected_tags()
+        filters.append(current_pane)
+        vtree = self.req.get_tasks_tree(name=current_pane, refresh=False)
+        # Re-applying search if some search is specified
+        search = self.search_entry.get_text()
+        if search:
+            filters.append(SEARCH_TAG)
+        # only reseting filters if applieds filters are different from current
+        # ones, leaving a chance for liblarch to make the good call on wether
+        # to refilter or not
```

Comment:
Not that important but somewhat annoys me:  
reseting → resetting  
applieds → the applied  
wether → whether

Reply:
Comments are important :) Fixed !